yarn-project/p2p/src/test-helpers/get-ports.ts

```diff
@@ -0,0 +1,8 @@
+import getPort from 'get-port';
+
+/**
+ * Get a list of ports for a given number of peers
+ * @param numberOfPeers - The number of peers to get ports for
+ * @returns A list of ports
+ */
+export const getPorts = (numberOfPeers: number) => Promise.all(Array.from({ length: numberOfPeers }, () => getPort()));
```

Comment:
Heads up that get-port [can cause issues](https://www.npmjs.com/package/get-port#user-content-beware) when run from multiple processes (as in CI, where we run using `parallel` or multiple jest workers). If we start seeing issues about ports in use, we should go back to this.

What I've used elsewhere is to just use port 0 for binding, so the OS allocates something for you, and then returning it. But I'm not sure that works with the current workflow.